### PR TITLE
add logs window

### PR DESCRIPTION
**Pull request by joostinyi**
- adds logs call out and window pop up upon `truss watch`
https://www.loom.com/share/9d8dd9e285fd45ba8926be471b005d40?sid=1b366d2c-a353-49bd-97a5-2a3b164ba458

### CODE REVIEW

File: truss/cli/cli.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import logging
 import os
 import sys
+import webbrowser
```
Comment (bolasim) [inline truss/cli/cli.py:5]: do we need to add this reuqiment to pyproject.toml? Where's it coming from?

File: truss/remote/baseten/remote.py [hunk 2]
```diff
@@ -70,6 +70,18 @@ def push(  # type: ignore
             truss_handle=truss_handle,
         )
 
+    def get_remote_logs_url(
+        self,
+        model_name: str,
+    ) -> str:
+        model = BasetenApi.get_model(self._api, model_name)
+        model_id = model["model_version"]["oracle"]["id"]
```
Comment (bolasim) [inline truss/remote/baseten/remote.py:78]: https://github.com/basetenlabs/truss/pull/512/files#diff-7d09bff19608e397354406c797171d6c5d7e45888c1d12e81e277f0a61a34598R145

**Review by bolasim — APPROVED** [review]
okay to merge. I don't love our graphql query stuff right now. Let's make a clean up ticket

File: truss/cli/cli.py (hunk 1 above)
Comment (joostinyi) [reply truss/cli/cli.py:5]: part of the python standard lib https://docs.python.org/3/library/webbrowser.html

**Comment by joostinyi** [comment]
> okay to merge. I don't love our graphql query stuff right now. Let's make a clean up ticket

agreed - thegql queries are pretty error prone as they are

**Pull request merged**